pkg/pgmodel/cache/series_cache.go

```diff
@@ -24,10 +24,13 @@ import (
 //this seems like a good initial size for /active/ series. Takes about 32MB
 const DefaultSeriesCacheSize = 250000
 
-const GrowCheckDuration = time.Minute //check whether to grow the series cache this often
+const GrowCheckDuration = time.Minute // check whether to grow the series cache this often
 const GrowEvictionThreshold = 0.2     // grow when evictions more than 20% of cache size
 const GrowFactor = float64(2.0)       // multiply cache size by this factor when growing the cache
 
+// SeriesCache is a cache of model.Series entries.
+//
+// GetSeriesFromProtos returns a model.Series entry given a list of Prometheus prompb.Label
```

Comment:
It's not out of place in the sense that I did purposefully put it there. Apparently one convention for documenting the methods of an interface is to put them in the comment _above_ the interface. The other convention is to directly document each method.

In this case, I only documented one method of the interface, which looks kinda weird.

What would you prefer I do?

Reply:
I guess that's true... However in this example, that comment line doesn't even contribute anything IMO. The description can be seen from function definition.

Suggestion, either document all (which I don't see a need for) or none since the interface is pretty self-explanatory.